frontend/views/components/Graphs/Bars.vue

```diff
@@ -12,38 +12,47 @@
     g(:transform='`translate(0,${middle})`')
       // Animate using scale from the middle
       g.g-animate(:style='`transform: scale3d(1,${ ready ? 1 : 0 },1)`')
+        // Surplus line on top of bars
+        g(
+          v-for='scaleLine in createScale'
+          :transform='`translate(0,${-scaleLine.position})`'
+        )
+          line.g-animate.g-animate-delay(
+            :style='`transform: scale3d(${ready ? 1 : 0},1,1)`'
+            x1='0'
+            y1='0'
+            :x2='ratioX'
+            y2='0'
+            stroke='#dbdbdb'
+            stroke-width='1'
+            stroke-dasharray='1'
+          )
+
         g.graph-bar(
           v-for='(member, index) in members'
           :transform='positionX(index)'
+          @mouseover='showLabel(index, member)'
+          @mouseleave='showLabel(index)'
         )
           // Total needed or total pledge bars
           path(
             v-for='(values, index) in [member.total, member.amount]'
             :class='color(values, index)'
-            :d='roundedRect(values >= 0, 0, positionY(values), width, height(values), width > 15 ? 3 : 1)'
+            :d='roundedRect(values >= 0, index === 1 && member.total !== Math.round(member.amount), 0, positionY(values), width, height(values), width > 15 ? 3 : 1)'
           )
 
       // Base with $0 on top of bars
       line(x1='0' y1='0' :x2='ratioX' y2='0' stroke='#dbdbdb' stroke-width='1')
 
-      // Surplus line on top of bars
-      g(
-        v-for='scaleLine in createScale'
-        :transform='`translate(0,${-scaleLine.position})`'
-      )
-        line.g-animate.g-animate-delay(
-          :style='`transform: scale3d(${ready ? 1 : 0},1,1)`'
-          x1='0'
-          y1='0'
-          :x2='ratioX'
-          y2='0'
-          stroke='#dbdbdb'
-          stroke-width='1'
-          stroke-dasharray='1'
-        )
+  .c-tag-user(
+    :class='{ positif: barAmount >= 0 }'
+    :style='{ opacity: barTotal || barAmount ? 1 : 0, transform: `translate3d(${labelX}px,${labelY}px,0)`} '
+  )
+    .c-tag-total {{currency(Math.abs(Math.round(barTotal)))}}
+    .c-tag-amount(v-if='Math.round(barAmount) !== Math.round(barTotal)') {{currency(Math.abs(Math.round(barAmount)))}}
 
-  .tag.mincome(:style='middleTag') {{ base }}
-  .tag.g-animate-opacity.g-animate-delay(
+  .c-tag.mincome(:style='middleTag') {{ base }}
+  .c-tag.g-animate-opacity.g-animate-delay(
     v-for='scaleLine in createScale'
     :style='{ opacity: ready ? 1 : 0, transform: "translate(0," + (-scaleLine.position + middle - labelPadding) + "px)" }'
```

Comment:
I noticed now that the last bar gets overlapped by the label. An easy fix by changing the transform X above, from 0 to 1px
![image](https://user-images.githubusercontent.com/14869087/72200731-e0abd700-3444-11ea-918f-ab4e3fd4db5c.png)